src/test/java/com/google/sps/JobStoreCenterTest.java

```diff
@@ -0,0 +1,332 @@
+    
+    Key projectKey = KeyFactory.createKey("Project", PROJECT1);
+    Entity actualProject = datastore.get(projectKey);
+
+    Assert.assertEquals(TIME2, actualProject.getProperty("lastAccessed"));
+  }
+
+  @Test
+  public void addProjectWithRunningJob() throws IOException, EntityNotFoundException {
+    // Add a project that has only one job (the job is running)
+    Map<String, List<JobJSON>> mapOfJobs = new HashMap<>();
+    mapOfJobs.put("europe-west1", Arrays.asList(JOB1));
+
+    ProjectLoader projectLoader = new ProjectLoaderStub(mapOfJobs, PROJECT1);
+    clock.setTime(TIME2);
+    jobStoreCenter.dealWithProject(PROJECT1, projectLoader);
+    
+    // Datastore should contain only 2 elements
+    int expectedNumberOfObjects = 2;
+    int actualNumberOfObjects = datastore.prepare(new Query()).countEntities();
+    Assert.assertEquals(expectedNumberOfObjects, actualNumberOfObjects);
+
+    int expectedNumberOfProjects = 1;
+    int actualNumberOfProjects = datastore.prepare(new Query("Project")).countEntities();
+    Assert.assertEquals(expectedNumberOfProjects, actualNumberOfProjects);
+
+    int expectedNumberOfJobs = 1;
+    int actualNumberOfJobs = datastore.prepare(new Query("RunningJob")).countEntities();
+    Assert.assertEquals(expectedNumberOfJobs, actualNumberOfJobs);
+  }
+
+  @Test
+  public void addProjectWithFinalisedJob() throws IOException, EntityNotFoundException {
+    // Add a project that has only one job (the job is finalised)
+    Map<String, List<JobJSON>> mapOfJobs = new HashMap<>();
+    mapOfJobs.put("europe-west2", Arrays.asList(JOB2));
+
+    ProjectLoader projectLoader = new ProjectLoaderStub(mapOfJobs, PROJECT1);
+    clock.setTime(TIME2);
+    jobStoreCenter.dealWithProject(PROJECT1, projectLoader);
+    
+    // Datastore should contain only 2 elements
+    int expectedNumberOfObjects = 2;
+    int actualNumberOfObjects = datastore.prepare(new Query()).countEntities();
+    Assert.assertEquals(expectedNumberOfObjects, actualNumberOfObjects);
+
+    int expectedNumberOfProjects = 1;
+    int actualNumberOfProjects = datastore.prepare(new Query("Project")).countEntities();
+    Assert.assertEquals(expectedNumberOfProjects, actualNumberOfProjects);
+
+    int expectedNumberOfJobs = 1;
+    int actualNumberOfJobs = datastore.prepare(new Query("FinalisedJob")).countEntities();
+    Assert.assertEquals(expectedNumberOfJobs, actualNumberOfJobs);
+  }
+
+  @Test
+  public void addJobWithBothTypesOfJobs() throws IOException, EntityNotFoundException {
+    // Add a project that has only one job (the job is finalised)
+    Map<String, List<JobJSON>> mapOfJobs = new HashMap<>();
+    mapOfJobs.put("europe-west1", Arrays.asList(JOB1));
+    mapOfJobs.put("europe-west2", Arrays.asList(JOB2));
+
+    ProjectLoader projectLoader = new ProjectLoaderStub(mapOfJobs, PROJECT1);
+    clock.setTime(TIME2);
+    jobStoreCenter.dealWithProject(PROJECT1, projectLoader);
+    
+    // Datastore should contain only 2 elements
```

Comment:
I think this is an old comment/needs updating?

Reply:
Yes, you are right. Thanks :D

Reply:
Deleted in https://github.com/googleinterns/step229-2020/pull/67/commits/976407ef6850b720819d5c8fe5f3c54436ec838f